redisdb/changelog.d/19696.added

```diff
@@ -0,0 +1 @@
+redisdb: add additional memory metrics
```

Comment:
```suggestion
Add additional memory metrics.
```
Very small changelog suggestion- this goes on the integration changelog so there is no need to include the integration name